Modules/OperationValidation/OperationValidation.psm1

```diff
@@ -213,17 +224,42 @@ param (
                                 # now determine if there's a diagnostics directory, or a version
                                 if ( test-path -path ($modDir.FullName + "\Diagnostics"))
                                 {
-                                    $modDir.FullName
-                                    break
+                                    # Did we specify a specific version to find?
+                                    if ($PSBoundParameters.ContainsKey('Version'))
+                                    {
+                                        $manifestFile = Get-ChildItem -Path $modDir.FullName -Filter "$modDir.psd1" | Select-Object -First 1
```

Comment:
>$modDir [](start = 102, length = 7)

when I see write things like this, i usually like to use `"${modDir}.psd1"` just to avoid confusion